cypress/support/commands.js

```diff
@@ -23,3 +24,31 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', () => {
+  cy.visit('/');
+  cy.get('.navbar-nav a:contains(Login)').click();
+  cy.get('#email').type(Cypress.env('user'));
+  cy.get('#password').type(Cypress.env('password'));
+  cy.get('button[type="submit"]').click();
+})
+
+Cypress.Commands.add('createsANote', data => {
+  cy.contains('Create a new note', { timeout: 30000 }).should('be.visible').click();
```

Comment:
Why are you overriding the default timeout here?

Reply:
After the application fills the login form and clicks on the submit button, it's been taking more than the default time of 4000ms and I get a Timeout error before I get redirected to the main page. Meanwhile, I figured out and added an assertion that waited for the new page to be loaded. I'll update in a new commit